server/routes/auth.js

```diff
@@ -1,18 +1,91 @@
 const express = require('express');
 const router = express.Router();
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
```

Comment:
Are these new packages from NPM? If so, are you missing a commit to your `package.json`.

EDIT: Oh, they were already added back in 442009b ✅